packages/rocketchat-ui/lib/chatMessages.coffee

```diff
@@ -378,8 +378,9 @@ class @ChatMessages
 		else if k is 75 and ((navigator?.platform?.indexOf('Mac') isnt -1 and event.metaKey and event.shiftKey) or (navigator?.platform?.indexOf('Mac') is -1 and event.ctrlKey and event.shiftKey))
 			RoomHistoryManager.clear rid
 
-	valueChanged: (rid, event) ->
-		this.determineInputDirection()
+	valueChanged: (rid, event, input) ->
+		if this.input.value.length is 1
```

Comment:
I tested here and worked, why do you think it wont work? also the derermineInputDirection() method uses this.input too.